src/main/java/org/opensearch/security/authtoken/jwt/JwtVendor.java

```diff
@@ -105,13 +105,16 @@ public String createJwt(
         List<String> roles,
         List<String> backendRoles
     ) throws Exception {
+        String tokenIdentifier = "obo";
```

Comment:
Can this variable be called `tokenType`?

Sometimes JWTs include a claim called `jti` that is a unique identifier for a token. In my experience, systems use `jti` to invalidate tokens since this identifier uniquely identifies a token.